refactor: Refactoring Large File Optimization

syuzanvs opened the pull request:
Split aiagentservice.ts and aiagentui.ts into smaller, focused modules to improve maintainability and enforce single responsibility principle.

Split files:
1. aiagentservice.ts ->
- moderate-content.ts: Content moderation logic
- prompt.ts: Prompt generation and handling
- check-model.ts: Model validation
- process-content.ts: Content processing utilities

2. aiagentui.ts ->
- ai-agent-button.ts: AI command button UI
- ai-agent-tone-button.ts: Tone selection UI
- ai-agent-ui-schema.ts: Schema registration

Closes #129

jjroelofs reviewed (changes requested):
It's not merging after I merged the tone of voice indicator PR, can you please fix merge conflicts?

jjroelofs reviewed (changes requested):
pls resolve conflicts, re-request my review and I'll look at it asap

syuzanvs committed:
refactor: Refactoring Large File Optimization

Split aiagentservice.ts and aiagentui.ts into smaller, focused modules to improve maintainability and enforce single responsibility principle.

Split files:

1. aiagentservice.ts ->
    - moderate-content.ts: Content moderation logic
    - prompt.ts: Prompt generation and handling
    - check-model.ts: Model validation
    - process-content.ts: Content processing utilities
2. aiagentui.ts ->
    - ai-agent-button.ts: AI command button UI
    - ai-agent-tone-button.ts: Tone selection UI
    - ai-agent-ui-schema.ts: Schema registration

Closes #129

jjroelofs reviewed (approved):
awesome, thanks!
